kbase-extension/static/kbase/templates/data_staging/ftp_file_table.html

```diff
@@ -17,7 +17,7 @@
             <td>{{mtime}}</td>
             <td>
                 {{#if imported }}
-                <div class="pull-right">
+                <div class="pull-right" style="margin-left: -5px">
```

Comment:
I'm on the fence about this, too, @ialarmedalien. Mostly, I'm lax enough to say that a single inline style change is probably ok. But I can also see the value in having zero inline styling, or at least not including new ones and eventually removing existing ones.

Reply:
I think we should aim for consistency. Most of the application has doesn't use inline styling.

Reply:
This whole section is going to be removed anyway as part of the redesign, just want to make sure that it doesn't break upon resize. So I don't think it really matters right now.... 

It doesn't seem worth it to me to make 10 new css elements and clutter up the existing css files, but let me know how you think I should proceed

Reply:
I think it's fine if we're planning to rewrite / reorganize everything anyway.